build failure against ghc-8.10: src/System/Plugins/Env.hs:81:32: error: Module ‘SysTools’ does not export ‘initLlvmConfig’

Tried to build `plugins` against `ghc-8.10`. Fails as:

```
Building library for plugins-1.6.1..
[ 5 of 12] Compiling System.Plugins.Env ( src/System/Plugins/Env.hs, dist/build/System/Plugins/Env.o, dist/build/System/Plugins/Env.dyn_o )

src/System/Plugins/Env.hs:81:32: error:
    Module ‘SysTools’ does not export ‘initLlvmConfig’
   |
81 | import SysTools (initSysTools, initLlvmConfig)
   |                                ^^^^^^^^^^^^^^
```

--- a/src/System/Plugins/Env.hs
+++ b/src/System/Plugins/Env.hs
@@ -78,7 +78,13 @@ import DynFlags (
   Way(WayDyn), dynamicGhc, ways,
 #endif
   defaultDynFlags, initDynFlags)
-import SysTools (initSysTools, initLlvmConfig)
+import SysTools ( initSysTools
+#if MIN_VERSION_ghc(8,10,1)
+                , lazyInitLlvmConfig
+#else
+                , initLlvmConfig
+#endif
+  )
 
 import Distribution.Package hiding (
 #if MIN_VERSION_ghc(7,6,0)
@@ -104,6 +110,11 @@ import Data.List.Split
 
 import qualified Data.Map as M
 import qualified Data.Set as S
+
+#if !MIN_VERSION_ghc(8,10,1)
+lazyInitLlvmConfig = initLlvmConfig
+#endif
+
 --
 -- and map Data.Map terms to FiniteMap terms
 --
@@ -493,10 +504,10 @@ lookupPkg' p = withPkgEnvs env $ \fms -> go fms p
                 -- real packages.
 #if MIN_VERSION_ghc(8,8,1)
                 settings <- initSysTools (libdir)
-                llvmConfig <- initLlvmConfig (libdir)
+                llvmConfig <- lazyInitLlvmConfig (libdir)
 #else
                 settings <- initSysTools (Just libdir)
-                llvmConfig <- initLlvmConfig (Just libdir)
+                llvmConfig <- lazyInitLlvmConfig (Just libdir)
 #endif
                 dflags <- initDynFlags $ defaultDynFlags settings llvmConfig
                 libs <- mapM (findHSlib

--- a/src/System/Plugins/Load.hs
+++ b/src/System/Plugins/Load.hs
@@ -104,7 +104,13 @@ import GHC                      ( defaultCallbacks )
 #else
 import DynFlags                 (defaultDynFlags, initDynFlags)
 import GHC.Paths                (libdir)
-import SysTools                 (initSysTools, initLlvmConfig)
+import SysTools                 ( initSysTools
+#if MIN_VERSION_ghc(8,10,1)
+                                , lazyInitLlvmConfig
+#else
+                                , initLlvmConfig
+#endif
+                                )
 #endif
 import GHC.Ptr                  ( Ptr(..), nullPtr )
 #if !MIN_VERSION_ghc(7,4,1)
@@ -120,6 +126,10 @@ import System.IO                ( hFlush, stdout )
 #endif
 import System.IO                ( hClose )
 
+#if !MIN_VERSION_ghc(8,10,1)
+lazyInitLlvmConfig = initLlvmConfig
+#endif
+
 ifaceModuleName = moduleNameString . moduleName . mi_module
 
 readBinIface' :: FilePath -> IO ModIface
@@ -128,10 +138,10 @@ readBinIface' hi_path = do
 #if MIN_VERSION_ghc(7,2,0)
 #if MIN_VERSION_ghc(8,8,1)
     mySettings <- initSysTools (libdir) -- how should we really set the top dir?
-    llvmConfig <- initLlvmConfig (libdir)
+    llvmConfig <- lazyInitLlvmConfig (libdir)
 #else
     mySettings <- initSysTools (Just libdir) -- how should we really set the top dir?
-    llvmConfig <- initLlvmConfig (Just libdir)
+    llvmConfig <- lazyInitLlvmConfig (Just libdir)
 #endif
     dflags <- initDynFlags (defaultDynFlags mySettings llvmConfig)
     e <- newHscEnv dflags

